_data/events.yml

```diff
@@ -69,25 +69,49 @@
 
 
 - name: "Bootcamp"
-  subTitle: ""
+  subTitle: "Home Automation"
+  by: "Mike"
   date: 2025-03-28
-  hide: true
   track: "Bootcamp"
   summary: >
-
+    The itenium bootcamp this year was all about "Home Automation". 
+    We kicked things off, as tradition goes, with breakfast, 
+    followed by the legendary "SMART HOUSE - REMA 1000" clip, 
+    a perfect reminder that maybe not everything should be 
+    automated.
+    <br><br>
+    Then it was time to get hands-on: teams like the Burnt 
+    Fingers Club and The Sparkonauts brought the heat -- literally, 
+    as they went about soldering their own smart sockets.
+    <br><br>
+    After dinner, we took things up a notch as we hooked the 
+    sockets and our sensor-loaded mini-houses to Home Assistant 
+    via one of the Raspberries. From the Cloud Cottage or 
+    The Overengineered Hut, everyone got a taste of smart living.
+    <br><br>
+    From solder fumes to smart rooms -- fueled by good vibes, 
+    snacks, and a closing reception. Till next year! 
   level: Introduction
   content: "Hands-On"
   type: ["On-Site"]
   duration: "8h"
   video: ""
-  image: ""
+  image: "assets/images-events/Bootcamp-2025-1.jpg"
   gallery:
-    - ""
-  linkedIn: ""
-  instagram: ""
-  twitter: ""
-  githubUrl: ""
-  blogPost: ""
+    - img: "assets/images-events/Bootcamp-2025-2.jpg"
+      alt: "A neatly arranged set of hand tools on a gray tiled surface, including screwdrivers, pliers, wrenches, utility knives, scissors, and a crimping tool—ready for various repair, construction, or DIY tasks."
```

Comment:
There is a difference between 'alt' and 'title':

alt: that could be just describing what the picture is
title: we're more interested in this -- it should be something specific to the picture, not something generic

Typically there is just an "alt" which is then used for "alt" and "title" which should be something specific (ie: itenium)